src/Libraries/RevitNodes/Elements/InternalUtilities/ElementQueries.cs

```diff
@@ -10,6 +10,31 @@ namespace Revit.Elements.InternalUtilities
     [IsVisibleInDynamoLibrary(false)]
     public static class ElementQueries
     {
+        private static readonly Dictionary<Type, string> ClassFilterExceptions = new Dictionary<Type, string>
```

Comment:
Correct. It's only stored for lookup purposes, but `HashSet` vs `Dictionary` contains tests are both equally fast. There is no advantage to using one over the other. http://stackoverflow.com/questions/2728500/hashsett-versus-dictionaryk-v-w-r-t-searching-time-to-find-if-an-item-exist